interceptors/client_test.go

```diff
@@ -329,7 +329,9 @@ func (s *ClientInterceptorTestSuite) TestBiStreamingReporting() {
 			if err == io.EOF {
 				break
 			}
-			require.NoError(s.T(), err, "reading pingStream shouldn't fail")
```

Comment:
It is incorrect to use `require` in a goroutine other than the main test goroutine.